Dialog/RoomConfigDialog.cpp

```diff
@@ -548,3 +548,27 @@ void RoomConfigDialog::on_spinBox_Layer0MappingType_valueChanged(int arg1)
         ui->CheckBox_Layer0Alpha->setEnabled(false);
     }
 }
+
+/// <summary>
+/// Slot function for spinBox_RasterType.
+/// </summary>
+/// <param name="arg1">
+/// The spinbox value.
+/// </param>
+void RoomConfigDialog::on_spinBox_RasterType_valueChanged(int arg1)
+{
+    switch(arg1)
+    {
+    case 0x00: ui->label_CurRasterType->setText("No Raster type effect"); break;
+    case 0x01: ui->label_CurRasterType->setText("Layer 3 Water effect 1"); break;
+    case 0x02: ui->label_CurRasterType->setText("Layer 3 Water effect 2"); break;
+    case 0x03: ui->label_CurRasterType->setText("Layer 0 Fog effect"); break;
+    case 0x04: ui->label_CurRasterType->setText("Layer 3 Fire effect 1"); break;
+    case 0x05: ui->label_CurRasterType->setText("Layer 3 Fire effect 2"); break;
+    case 0x06: ui->label_CurRasterType->setText("Layer 3 DOUBLE-SCR(A): top AUTO(1/8),bottom(non)"); break;
+    case 0x07: ui->label_CurRasterType->setText("Layer 3 DOUBLE-SCR(A): top(non),bottom AUTO(1/8)"); break;
+    case 0x08: ui->label_CurRasterType->setText("alpha Fire effect 1"); break;
+    case 0x09: ui->label_CurRasterType->setText("alpha Fire effect 2"); break;
+    default: ui->label_CurRasterType->setText("Undefined");
```

Comment:
Should it be possible for the default to get triggered? If not, let's use 'singleton->GetOutputWidgetPtr()->PrintString(message);' to print a warning to the console. Even if it is in a modal dialog, it will appear after in the main window and indicate that some unintended behavior occurred.

Reply:
has been discuessed and don't need.